github_connector/models/github_repository_branch.py

```diff
@@ -192,7 +192,7 @@ def _download_code(self):
                     res = check_output(
                         ["git", "pull", "origin", branch.name], cwd=branch.local_path
                     )
-                    if branch.state == "to_download" or b"up-to-date" not in res:
+                    if branch.state == "to_download" or b"up to date" not in res:
```

Comment:
`to_download` is correct as well?

Reply:
`to_download`' status is not necessary because it is not possible to be in that status if the repository has already been cloned.